packages/headless/src/controllers/facets/facet/headless-facet.ts

```diff
@@ -46,53 +46,69 @@ import {FacetValueState} from '../../../features/facets/facet-api/value';
 export {FacetOptions, FacetSearchOptions, FacetValueState};
 
 export interface FacetProps {
-  /** The options for the `Facet` controller. */
+  /**
+   * The options for the `Facet` controller.
+   * */
   options: FacetOptions;
 }
 
 /**
  * The `Facet` headless controller offers a high-level interface for designing a common facet UI controller.
  */
 export interface Facet extends Controller {
+  /**
+   * Provides methods to search the facet's values.
+   */
   facetSearch: FacetSearch;
 
   /**
    * Toggles the specified facet value.
-   * @param selection The facet value to toggle.
+   *
+   * @param selection - The facet value to toggle.
    */
-  toggleSelect: (selection: FacetValue) => void;
+  toggleSelect(selection: FacetValue): void;
 
   /**
    * Checks whether the specified facet value is selected.
-   * @param value The facet value to check.
+   *
+   * @param value - The facet value to check.
    * @returns Whether the specified facet value is selected.
    */
-  isValueSelected: (value: FacetValue) => boolean;
+  isValueSelected(value: FacetValue): boolean;
 
-  /** Deselects all facet values.*/
-  deselectAll: () => void;
+  /**
+   * Deselects all facet values.
+   * */
+  deselectAll(): void;
 
-  /** Sorts the facet values according to the specified criterion.
-   * @param criterion The criterion to sort values by.
+  /**
+   * Sorts the facet values according to the specified criterion.
+   *
+   * @param criterion - The criterion to sort values by.
    */
-  sortBy: (criterion: FacetSortCriterion) => void;
+  sortBy(criterion: FacetSortCriterion): void;
```

Comment:
When writing `sortBy:`, api-extractor (correctly) marks the member as a property, whereas `sortBy(...)` is marked as a method. This affects how the doc-site displays the entry. It may be possible to correct for this in the parser, but for now I am marking it as a limitation.

We should write methods using method notation so that they display under the "Methods" section in the reference doc.

Reply:
As a developer, I. think it's going to be tricky to follow all these rules correctly, especially if you're new to the project

Reply:
Yeah... 🤔  

Why does it matter that it appear as a function prop versus a method ? 

Both are pretty much exactly the same thing.
Is it just so it appears "nicely" organized ... ?

Reply:
I agree that rules should be at a minimum. Ideally the parser handle the complexity behind the scenes. But yes, for what we have now, the distinction helps organize the documentation nicely.

My goal is to have a first version of the documentation up by next Monday, and continue improving from there.